src/modules/unleash.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Please fix the identing in this file

Reply:
Fixed on https://github.com/HathorNetwork/hathor-wallet/pull/589/commits/b08a92e96d9e300b35a288f5cdf5a129095651dd